docs/apis/avalanchego/apis/ipc.md

```diff
@@ -50,6 +50,13 @@ This API uses the `json 2.0` RPC format.
 
 ### `ipcs.publishBlockchain`
 
+
+:::caution
+
+Deprecated as of **v1.9.12**.
```

Comment:
Please link to the release note of v1.9.12 in all places. Thanks.

Reply:
Sure. Fixed ed165cc788077c48e3527b7d9c1fc65e09516c23